lib/quadquizaminos_web/router.ex

```diff
@@ -39,7 +39,8 @@ defmodule QuadquizaminosWeb.Router do
     live "/leaderboard", LeaderboardLive
     live "/leaderboard/:board_id", LeaderboardLive.Show
     live "/contestboard", ContestboardLive
-    live "/contests", ContestsLive
+    live "/contests", ContestsLive, :index
+    live "/contests/:contest_id", ContestsLive, :show
```

Comment:
Lets use the param `id` instead of `contest_id` this is because we can already infer from the url that the id belongs to the contest